app/stores/DocumentsStore.ts

```diff
@@ -753,6 +753,23 @@ export default class DocumentsStore extends BaseStore<Document> {
     await star?.delete();
   };
 
+  subscribe = async (document: Document) => {
+    await this.rootStore.subscriptions.create({
+      documentId: document.id,
+      // REVIEW: Seems like a good location
+      // to describe event for now.
+      event: "documents.update",
+    });
+  };
+
+  unsubscribe = async (document: Document) => {
+    const subscription = this.rootStore.subscriptions.orderedData.find(
+      (subscription) => subscription.documentId === document.id
+    );
+
+    await subscription?.delete();
+  };
```

Comment:
Should this also check for `subscription.userId === currentUser.id`? Probably being paranoid.

Reply:
Not paranoid, good idea. Seems v possible we'll have other users subscriptions in this store later